app/models/concerns/rad_security_role.rb

```diff
@@ -61,4 +62,10 @@ def validate_rules
       errors.add(:allow_invite, 'is not applicable') if allow_invite? && RadConfig.disable_invite?
       errors.add(:allow_sign_up, 'is not applicable') if allow_sign_up? && RadConfig.disable_sign_up?
     end
+
+    def validate_external_security_role
+      return unless external && !RadConfig.external_users?
```

Comment:
we like using the question mark version of the field name for all boolean fields, like `external?`